src/api-helper/listener/executeReview.ts

```diff
@@ -81,7 +81,15 @@ export function executeReview(projectLoader: ProjectLoader,
                         review,
                     };
                     sendErrorsToSlack(reviewerErrors, addressChannels);
-                    const reviewResponses = await Promise.all(reviewListeners.map(l => l(rli)));
+                    const reviewResponses = await Promise.all(reviewListeners.map(async l => {
+                        try {
+                            return l.listener(rli);
+                        } catch (err) {
+                            await rli.addressChannels(`:crying_cat_face: Review listener '${l.name}' failed: \n\`\`\`${
```

Comment:
i think we should keep the stack trace for the logs.

(not blocking)

Reply:
Good point